packages/modules/src/modules/banners/momentTemplate/MomentTemplateBanner.tsx

```diff
@@ -225,7 +277,7 @@ const styles = {
             // Mobile Sticky Header Styles
             background: ${background};
             position: sticky;
-            top: 140px; // 140px for banner visual
+            top: ${choiceCards ? '0px' : '140px'}; // 140px for banner visual
```

Comment:
140px offset from the top for banner visual on mob, not needed for choice cards, as they will sit at the bottom of the container on mob